### PR TITLE
Adding deploymnet.json to enable CI

### PR DESCRIPTION
**Pull request by tellesnobrega**
Enabling CI for this repo.

### CODE REVIEW

**Review by FelipeNascimento — APPROVED** [review]

File: .vtex/deployment.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "description": "The repository for mapped and implemented purchase testing scenarios for the Checkout UI",
+    "build": {
+      "provider": "jenkins",
+      "type": "docker-ecr",
+      "parameters": {
+        "AWS_ACCOUNT_ID": "558830342743",
+        "AWS_REGION": "us-east-1",
+        "IMAGE_NAME": "healthcheck/webtests/checkout",
+        "SLACK_CHANNEL": "jenkins-notifications",
+        "DOCKER_FILE": "dockerfiles/stable"
```
Comment (FelipeNascimento) [inline .vtex/deployment.json:12]: It's a directory and must to have the full path of dockerfile

**Review by FelipeNascimento — CHANGES_REQUESTED** [review]
Dockerfile path is invalid as CI said:
`Error response from daemon: unexpected error reading Dockerfile: read /var/lib/docker/tmp/docker-builder556778397/dockerfiles/stable: is a directory`

**Review by lucasecdb — APPROVED** [review]

**Review by wdsrocha — APPROVED** [review]

**Comment by wdsrocha** [comment]
anyone knows how to re-run the Jenkins CI?

**Review by wdsrocha — CHANGES_REQUESTED** [review]
Same as Felipe Nascimento pointed out

**Review by FelipeNascimento — APPROVED** [review]

**Review by wdsrocha — APPROVED** [review]

**Pull request merged**